docs/configure.md

```diff
@@ -0,0 +1,15 @@
+# skyWM - Configuration Documentation
+
+## Autostarting
+
+Add shell commands to autostart by adding them to the `AUTOSTART` array on `main.rs`.
+Ajusting the array size accordinly
+
+### E.g.
+```
+const AUTOSTART: [&str; 3] = [
```

Comment:
Requires information about how `const AUTOSTART: [&str; NUM] = [` has a number that needs to be changed.